docs/src/main/sphinx/installation/plugins.md

```diff
@@ -0,0 +1,283 @@
+# Plugins 
+
+Trino uses a plugin architecture to extend its capabilities and integrate with
+various data sources and other systems. Trino includes many plugins as part of
+the binary packages - specifically the `tar.gz` archive and the Docker
+container.
+
+The main plugins implement some the following functionalities:
```

Comment:
This page kinda duplicates https://trino.io/docs/current/develop/spi-overview.html#deploying-a-custom-plugin

Reply:
I will rejig that page as well. Forgot about the details we have there

Reply:
Rejigging done.